spark/src/main/scala/org/apache/spark/sql/delta/commands/WriteIntoDelta.scala

```diff
@@ -260,6 +261,24 @@ case class WriteIntoDelta(
               sparkSession.conf.get(DeltaSQLConf.REPLACEWHERE_DATACOLUMNS_WITH_CDF_ENABLED) &&
               cdcExistsInRemoveOp) {
             var dataWithDefaultExprs = data
+            // Add identity columns if they are not in `data`.
+            // Column names for which we will track identity column high water marks.
+            val trackHighWaterMarks = scala.collection.mutable.Set[String]()
```

Comment:
```suggestion
            val trackHighWaterMarks = scala.collection.mutable.Set.empty[String]
```
I'd import `scala.collection.mutable`.